### DISCUSSION TITLE
Update to 30.0.1

### DISCUSSION THREAD

**Discussion by g-work** [nextcloud/all-in-one#5514] [Questions] [answered]
I'm on Nextcloud 30.0.0 (AIO 9.7) and on latest channel. Shouldn't I get a notification that 30.0.1 is available? Or do I have to update manually? I'm aware that I can force the update via CLI but I thought i would get informed that an update is available. Am I wrong with this?

**Comment by JimmyKater** [comment]
Usually you should be informed on Saturdays if there's a new AiO Version available (starting with saturday right after the update became available). 
I would recommend to wait for the official update since you then can be sure that it's tested and that possible neccessary adjustments has been made.

**Reply by g-work** [reply]
> Usually you should be informed on Saturdays if there's a new AiO Version available

That's what I thought - but I didn't get a notification and also Nextcloud All in One Interface says "Your containers are up-to-date." Have you any idea what's wrong here?

**Reply by JimmyKater** [reply]
@g-work 
nothing's wrong afaik. It just wasn't published, yet. So hold your horses ;-)
Just wait for AIO v9.8.0 - it's already in the making.

**Reply by g-work** [reply]
Ah, okay, I thought there was an internal update mechanism. Does the Nextcloud version in an AIO installation depend directly on the AIO version? So there is no way to update to 30.0.1 at the moment. If that's the case, I'll indeed  just wait for the notification. Thanks for the clarification!

Edit: Just saw [here](https://github.com/nextcloud/all-in-one/discussions/5133) that you can force the update via internal updater. Does this even make sense in an AIO installation?

**Reply by g-work** [reply]
> Just wait for AIO v9.8.0 - it's already in the making.

Well, I'm on AIO 9.8.0 now, but still 30.0.0 while 30.0.2 is already out. I think I don't understand how updates are supposed to work in AIO.

**Comment by szaimen** [comment]
Hi, either you opted in during the initial installation to install nc30 or you upgraded to nc30 manually. In either case you need to upgrade nc manually via https://github.com/nextcloud/all-in-one/discussions/5133 as long as https://github.com/nextcloud/all-in-one/issues/5249 is not implemented and released.

**Reply by g-work** [reply]
Yes, I installed 30 in the beginning. Updated manually to 30.0.2 today, works great. Thanks for your support!

**Reply by szaimen** [reply, marked as answer]
Nc30 is now released with v10.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
